main.py

```
@@ -140,22 +139,53 @@ def mix(self, pcm: bytes, level=0.3) -> bytes:  # Default level reduced to 0.3
             out += chunk
         self.idx = idx % len(self.buffer)  # Update index only once
         
-        # Boost the original audio to make it clearer
+        # Using NumPy for audio mixing
         try:
```

We might wanna do this as fallback. If audioop isn't found, it can be manually done. I don't know if we want to get rid of audioop (instead of just falling back on the new code when audioop throws.